elide-async/pom.xml

```diff
@@ -101,6 +101,18 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+            <version>6.0.2.Final</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.el</groupId>
```

Comment:
What is this dependency for?

Reply:
I had to include this dependency for the constraint violation unit tests. Turns out  for Hibernate Validator I needed to provide the expression language dependency as well